Geek-Collectors-Network/geek-collectors-network
# 201 add user facing feedback to registration page

### Summary
- Added loading icon to "Sign Up' button
- Added pop-up icon telling user if they successfully registered or not

### Testing

1. Spin up resources
2. Navigate to `http://localhost:8080/regiister`
3. Create new user; confirm that (1) The `Sign Up` button has a spinner, (2) that a green pop-up at the bottom of the page appears, (3) that you're routed to the dashboard page
